src/components/InputPreview.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, useState } from "react";
+import { Button } from "./atomic";
+
+type PreviewInput = {
```

Comment:
Maybe `InputPreviewProps` would be nice (since InputPreview, PreviewInput look similar, and their relationship is not clear)

Reply:
+1 to @hcsong213's comment. Always follow the convention [ComponentName]Props

Also IconFileUploader is more descriptive/accurate than InputPreview. 

But also, is a component needed here? (It doesn't seem like many lines and idk if it will be reused)

Reply:
I was thinking to reuse the logo in settings later